README.md

```diff
@@ -2,6 +2,26 @@
 
 This is a Drupal module that contains the EC corporate entity types.
 
+The main entity type is the RDF Entity which has the following bundles:
+
+* Event
+* Announcement
+
+All bundles of this entity type have predicate mappings for RDF shared storage.
+In order to determine the provenance of an entity, a "provenance URI" can be set
+at the site level, which gets automatically saved with the entity. Access rules also
+depend on this URI.
+
+When installing the module, the provenance URI is set based on the base URL of the site.
+However, it can be overridden by adding the following line to the settings.php file (with the
+your respective URI):
+
+```
+$config['oe_content.settings']['provenance_uri'] = 'http://example.com';
+```
+
+The task runner will automatically add this on your development environment.
```

Comment:
The task runner does not actually set this, we should either add it there and keep this line or remove this line since this is not necessary anymore to have the module running.